Fix failing tests on windows

## 🐛 Bug description

```bash
 tests\ignite\contrib\metrics\test_precision_recall_curve.py:4: in <module>
    from sklearn.metrics import precision_recall_curve
c:\miniconda3\envs\test\lib\site-packages\sklearn\__init__.py:80: in <module>
    from .base import clone
c:\miniconda3\envs\test\lib\site-packages\sklearn\base.py:21: in <module>
    from .utils import _IS_32BIT
c:\miniconda3\envs\test\lib\site-packages\sklearn\utils\__init__.py:20: in <module>
    from scipy.sparse import issparse
c:\miniconda3\envs\test\lib\site-packages\scipy\__init__.py:104: in <module>
    from . import _distributor_init
c:\miniconda3\envs\test\lib\site-packages\scipy\_distributor_init.py:61: in <module>
    WinDLL(os.path.abspath(filename))
c:\miniconda3\envs\test\lib\ctypes\__init__.py:364: in __init__
    self._handle = _dlopen(self._name, mode)
E   OSError: [WinError 126] The specified module could not be found
```

Related to https://github.com/scipy/scipy/issues/12400


A fix can be downgrading scipy to `1.4.1`.

## Environment

 - PyTorch Version (e.g., 1.4):
 - Ignite Version (e.g., 0.3.0):
 - OS (e.g., Linux):
 - How you installed Ignite (`conda`, `pip`, source):
 - Python version:
 - Any other relevant information:

--- a/.github/workflows/unittests.yml
+++ b/.github/workflows/unittests.yml
@@ -38,8 +38,9 @@ jobs:
       shell: bash -l {0}
       run: |
         conda install pytorch torchvision cpuonly -c ${{ matrix.pytorch-channel }}
-        # Keep fix in case of problem with torchvision nightly releases
         pip install -r requirements-dev.txt
+        # Fixes #1153
+        pip install --upgrade scipy==1.4.1
         python setup.py install
 
     - name: Run Tests

